src/Appwrite/Usage/Calculators/TimeSeries.php

```diff
@@ -315,6 +316,7 @@ private function createOrUpdateMetric(string $projectId, string $time, string $p
                     'metric' => $metric,
                     'value' => $value,
                     'type' => $type,
+                    'region' => App::getEnv('_APP_REGION', 'default'),
```

Comment:
Same, will be better to pass this to the constructor so we don't couple the calculator with the class App.